astroid/brain/brain_typing.py

```diff
@@ -240,16 +240,17 @@ def _forbid_class_getitem_access(node: ClassDef) -> None:
     def full_raiser(origin_func, attr, *args, **kwargs):
         """
         Raises an AttributeInferenceError in case of access to __class_getitem__ method.
-        Otherwise just call origin_func.
+        Otherwise, just call origin_func.
         """
         if attr == "__class_getitem__":
             raise AttributeInferenceError("__class_getitem__ access is not allowed")
         return origin_func(attr, *args, **kwargs)
 
     try:
         node.getattr("__class_getitem__")
-        # If we are here, then we are sure to modify object that do have __class_getitem__ method (which origin is one the
-        # protocol defined in collections module) whereas the typing module consider it should not
+        # If we are here, then we are sure to modify object that do have
```

Comment:
While we are here, do we want to translate this into actual English? 😄

Reply:
What would you suggest ? 😄

Reply:
(Maybe it's because I'm french like the original writer but it make sense to me)

Reply:
I mean `to modify object` seems to be missing a `the`? or should it be `objects`? The tenses seem to be a bit messed up and honestly I don't even understand what it is trying to say 😄

Reply:
```suggestion
        # If we are here, then we are sure to modify objects that do have
```